src/pages/en/guides/markdown-content.md

```diff
@@ -1,139 +1,37 @@
 ---
 layout: ~/layouts/MainLayout.astro
 title: Markdown
-description: An intro to Markdown with Astro.
+description: Building Pages in Markdown
 ---
+Markdown content is commonly used for text-heavy pages like blog posts and documentation. Astro has built-in support for writing in Markdown.
```

Comment:
```suggestion
Markdown is a light weight formatting language that allows anyone to write documents and have them turned into html. Astro takes its built in support for Markdown further, enhancing the developer experience by letting you do more to your Markdown files. 
```

Reply:
I'll play with this. I personally don't think we have to explain what Markdown content *is*, but I do like emphasizing that you can do more with Markdown in Astro than just Markdown a la mdx.